src/Home.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
```

Comment:
Done, @meenakshi-dhanani . Now Home.jsx sits inside src directory and I removed the js directory. 😉